lib/Runtime/Library/DateImplementation.h

```diff
@@ -418,7 +451,14 @@ namespace Js {
             bs->AppendChars(_u(' '));
 
             // year is directly after day, month, daydigit for IE11+
-            bs->AppendChars(pymd->year, 10, ConvertLongToString);
+            if (pymd->year > 0)
```

Comment:
nit: wrap the `pymd->year` in parens for readability.

Reply:
Done.